### PR TITLE
fix built in plugin js

### LINKED ISSUE #3633: Checkboxes in built-in plugin configurations are broken in 10.6.0
Checkboxes in built-in plugin configurations are broken due to jQuery changes.

See jellyfin/jellyfin-plugin-kodisyncqueue#25 for needed fix.

### PR DESCRIPTION
**Pull request by crobibero**
Fixes #3633

### CODE REVIEW

File: MediaBrowser.Providers/Plugins/MusicBrainz/Configuration/config.html [hunk 1]
```diff
@@ -36,33 +36,35 @@
                 uniquePluginId: "8c95c4d2-e50c-4fb0-a4f3-6c06ff0f9a1a"
             };
 
-            $('.musicBrainzConfigPage').on('pageshow', function () {
-                Dashboard.showLoadingMsg();
-                ApiClient.getPluginConfiguration(MusicBrainzPluginConfig.uniquePluginId).then(function (config) {
-                    $('#server').val(config.Server).change();
-                    $('#rateLimit').val(config.RateLimit).change();
```
Comment (dmitrylyzo) [inline MediaBrowser.Providers/Plugins/MusicBrainz/Configuration/config.html:43]: I see no `change`-listeners here, so event emiting is not required, imo.

Comment (MrTimscampi) [reply MediaBrowser.Providers/Plugins/MusicBrainz/Configuration/config.html:43]: Note that the listeners may be on the web client side.

Comment (dmitrylyzo) [reply MediaBrowser.Providers/Plugins/MusicBrainz/Configuration/config.html:43]: Looks like there is one. `emby-input` changes label style on `change`.

Comment (crobibero) [reply MediaBrowser.Providers/Plugins/MusicBrainz/Configuration/config.html:43]: I think I've been removing the change() functions everywhere. Should they be kept or does it not matter?

Comment (dmitrylyzo) [reply MediaBrowser.Providers/Plugins/MusicBrainz/Configuration/config.html:43]: We could add `valueset` handler for `emby-input`.
But if we don't want to do this right now, we should keep event emitting.

To "perfectly" emulate `change` event:
```js
elem.dispatchEvent(new Event('change', {
    bubbles: true,
    cancelable: false
}));
```

**Pull request merged**